resources/i18n/cs_CZ/fdmprinter.def.json.po

```diff
@@ -1883,6 +1903,26 @@ msgctxt "magic_fuzzy_skin_outside_only description"
 msgid "Jitter only the parts' outlines and not the parts' holes."
 msgstr "Rozmazat jen okrajové části modelu a žádné díry modelu."
 
+msgctxt "variant_name"
+msgid "KLEMA 180 Size"
+msgstr ""
+
+msgctxt "variant_name"
+msgid "KLEMA 250 Pro Size"
+msgstr ""
+
+msgctxt "variant_name"
+msgid "KLEMA 250 Size"
+msgstr ""
+
+msgctxt "variant_name"
+msgid "KLEMA 250 Twin Size"
+msgstr ""
+
+msgctxt "variant_name"
+msgid "KLEMA 500 Size"
+msgstr ""
+
```

Comment:
Discussion results: We don't really want these in here, alter the 'variant-name' script to listen to a new 'should translate' (default = false) in fdmprinter.